templates/compose/siyuan.yaml

```
@@ -0,0 +1,28 @@
+# documentation: https://github.com/siyuan-note/siyuan
+# slogan: A privacy-first, self-hosted, fully open source personal knowledge management software, written in typescript and golang.
+# tags: note-taking,markdown,pkm
+# logo: svgs/siyuan.svg
+# port: 6806
+
+services:
+  siyuan:
+    image: b3log/siyuan
```

You should pin to a specific version ( v3.3.5 in this case )

```suggestion
    image: b3log/siyuan:v3.3.5
```